packages/flutter/lib/src/widgets/text.dart

```diff
@@ -201,6 +201,77 @@ class DefaultTextStyle extends InheritedTheme {
   }
 }
 
+/// The [TextHeightBehavior] that will apply to descendant [Text] widgets which
+/// have not explicitly set [Text.textHeightBehavior].
+///
+/// See also:
+///
+///  * [DefaultTextStyle], which defines a [TextStyle] to apply to descendant
+///    [Text] widgets.
+class DefaultTextHeightBehavior extends InheritedTheme {
+  /// Creates a default text height behavior for the given subtree.
+  ///
+  /// This will take precedence over [DefaultTextStyle.textHeightBehavior].
+  ///
+  /// The [textHeightBehavior] and [child] arguments are required and must not be null.
+  const DefaultTextHeightBehavior({
+    Key key,
+    @required this.textHeightBehavior,
+    @required Widget child,
+  }) :  assert(textHeightBehavior != null),
+        assert(child != null),
+        super(key: key, child: child);
+
+  /// A const-constructible default text height behavior that provides a
+  /// [TextHeightBehavior] of null.
+  ///
+  /// Returned from [of] when the given [BuildContext] doesn't have an enclosing
+  /// [DefaultTextHeightBehavior].
+  ///
+  /// This constructor creates a [DefaultTextStyle] that lacks a [child], which
+  /// means the constructed value cannot be incorporated into the tree.
+  const DefaultTextHeightBehavior.fallback({ Key key })
```

Comment:
I don't think this is needed; see related comments

Reply:
Removed.

Reply:
It seems to still be here.